assets/global.js

```diff
@@ -378,10 +378,17 @@ class ModalDialog extends HTMLElement {
       'click',
       this.hide.bind(this)
     );
+    this.querySelectorAll('img[data-media-id], [role="document"]').forEach(element => {
```

Comment:
These selectors make sense to me, but definitely open to other ideas.

Reply:
I wonder if we should then add `role="document"` to the video modal on the collage section so it behaves the same way. I think that would make sense for continuity in the behaviour. 

From what I can see only two `moda-dialog` elements are used, on `main-product.liquid` and `collage.liquid`.

Reply:
Let me know if this shouldn't be happen anyway and we can disregard my comment